web/src/components/footer.tsx

```diff
@@ -0,0 +1,86 @@
+//
+// DevHub
+//
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Jason Leach on 2018-09-26.
+//
+
+import styled from '@emotion/styled';
+import React from 'react';
+
+const StyledFooter = styled.footer`
+  background-color: #036;
+  border-top: 2px solid #fcba19;
+  color: #fff;
+  display: flex;
+  flex-flow: row wrap;
+  text-align: center;
+  position: fixed;
+  bottom: 0px;
+  height: 46px;
```

Comment:
fixing a height can run you into issues later. Its best to often just add padding to make the 'expected' height. For example if text wraps on screen resizes it can look wonky with a fixed height.